hs/app/reach/Main.hs

```diff
@@ -1112,13 +1105,7 @@ compile = command "compile" $ info f d
 
           (cd "$$HS" && make expand)
```

Comment:
I think we want
```suggestion
          (cd "$$HS" && make hs-build)
```
because `expand` is a prerequisite of `hs-build` and `stack exec` doesn't force a (re-)build.

Reply:
Go for it

Reply:
You own this branch now? :)